src/components/TodosList.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodosList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
Add a conditional rendering to hide the list and footer if there are no todos yet.